src/core/utils/modal.tsx

```diff
@@ -72,12 +72,12 @@ function Modal({
           classNames
         )}
         role="dialog"
-        aria-labelledby={`modal-${modalId}`}
+        aria-labelledby={`modal-${modalId}-description`}
         data-cy={dataCy}
       >
         <div
           className="modal__screen-reader-description"
-          id={`modal-${modalId}`}
+          id={`modal-${modalId}-description`}
```

Comment:
I am not sure I understand this, why is description added?

Reply:
That's just because it makes the ID more case specific, since it's used on the element that contains the description. But it's not the key change and I can revert it back if you don't think it makes sense.